src/vs/base/browser/markdownRenderer.ts

```diff
@@ -547,7 +547,7 @@ export function renderMarkdownAsPlaintext(markdown: IMarkdownString, withCodeBlo
 
 	const html = marked.parse(value, { async: false, renderer: withCodeBlocks ? plainTextWithCodeBlocksRenderer.value : plainTextRenderer.value }).replace(/&(#\d+|[a-zA-Z]+);/g, m => unescapeInfo.get(m) ?? m);
 
-	return sanitizeRenderedMarkdown({ isTrusted: false }, html).toString();
+	return sanitizeRenderedMarkdown({ isTrusted: false }, html).toString().replace(unescapeRegex, match => unescapeInfo.get(match) ?? match);
```

Comment:
Looks like there's already unescaping happening above: `.replace(/&(#\d+|[a-zA-Z]+);/g, m => unescapeInfo.get(m) ?? m);`

Do we still need both of these?

Reply:
Yes, based on my testing, it seems those are re-added at some point during the sanitization.